test/utils/structs/MerkleTree.test.js

```
@@ -53,128 +95,160 @@ describe('MerkleTree', function () {
+          .withArgs(leaves.at(-1), i, tree.root);
         await expect(this.mock.root()).to.eventually.equal(tree.root);
         await expect(this.mock.nextLeafIndex()).to.eventually.equal(BigInt(i) + 1n);
       }
     });
 
-    it('pushing to a full tree reverts', async function () {
-      await Promise.all(Array.from({ length: 2 ** Number(DEPTH) }).map(() => this.mock.push(ethers.ZeroHash)));
-
-      await expect(this.mock.push(ethers.ZeroHash)).to.be.revertedWithPanic(PANIC_CODES.TOO_MUCH_MEMORY_ALLOCATED);
+    it('tree is full', async function () {
+      for (const _ of range(2 ** DEPTH)) {
```

ESLint

[error] '_' is assigned a value but never used.

(no-unused-vars)